samples/ImplicitFlow01/AppWithFetchClient/wwwroot/app.js

```diff
@@ -0,0 +1,72 @@
+/* global window, fetch, Oidc */
+
+//
+// Define helper functions.
+//
+const fetchUserResources = (accessToken) => {
+  const url = `${window.location.origin}/api`;
+  const options = {
+    method: 'GET',
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  };
+
+  // See https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
+  fetch(url, options)
+    .then(response => response.json())
+    .then(obj => JSON.stringify(obj))
+    .then((json) => {
+      window.document.getElementById('fetch_response').innerText = json;
+    })
+    .catch(err => console.log(err)); // eslint-disable-line no-console
+};
+
+const displayUserData = (user) => {
+  const json = JSON.stringify(user, null, 4);
+  window.document.getElementById('signin_response').innerText = json;
+};
+
+const signinSilentError = (err) => {
+  window.document.getElementById('signin_response').innerText = err;
+};
+
+//
+// Configure the oidc-client.js UserManager.
+//
+const settings = {
+  authority: window.location.origin,
+  silent_redirect_uri: window.location.origin,
+  client_id: 'my-client',
+  response_type: 'id_token token',
```

Comment:
Is this underscore space combo on purpose?

Reply:
What do you mean by "underscore space combo?" 

Are you referring to the part that says, `"id_token token"`? If so, then yes, that is on purpose. It means that we're asking for both an ID Token and an Access Token, and that way of writing it is part of the OpenID Connect specification. 

If not, what do you mean?

Reply:
So the spec has a space delimiter?

Reply:
Yes.

Reply:
> Extension response types MAY contain a space-delimited (%x20) list of values...

https://tools.ietf.org/html/rfc6749#section-3.1.1